functional-tests/src/main/java/com/redhat/che/selenium/core/workspace/CheStarterWrapper.java

```diff
@@ -208,12 +212,16 @@ private void patchPrepareEnv(String workspaceName, String token) throws IOExcept
 
     try {
       response = client.newCall(request).execute();
-      if (response.isSuccessful()) {
+      boolean responseStatus = response.isSuccessful();
+      int responseCode = response.code();
+      ResponseBody responseBody = response.body();
+      response.close();
+      if (responseStatus) {
```

Comment:
Suggestion: rename to `responseSuccess`. Status could be confused with status code.

Reply:
noted :)